ui/src/app/workflows/components/workflow-logs-viewer/workflow-logs-viewer.tsx

```diff
@@ -289,11 +289,11 @@ export function WorkflowLogsViewer({workflow, initialNodeId, initialPodName, con
                 )}
                 {execSpec(workflow).podGC && (
                     <>
-                        <WarningIcon /> Your pod GC settings will delete pods and their logs{' '}
+                        <WarningIcon /> Your pod GC settings will delete pods{' '}
                         {execSpec(workflow).podGC.deleteDelayDuration ? `after ${execSpec(workflow).podGC.deleteDelayDuration}` : 'immediately'} on completion.
                     </>
                 )}{' '}
-                Logs may not appear for pods that are deleted.{' '}
+                Logs may not appear for pods that are deleted unless you have workflow archiving enabled.{' '}
```

Comment:
```suggestion
                Logs may not appear for pods that are deleted unless you have <a href="https://argo-workflows.readthedocs.io/en/latest/configure-archive-logs/>archive logs<\> enabled.{' '}
```

Archive Logs and and Workflow Archive are separate features (that may be used together).

Also added a link. Linter may want that formatting adjusted though